src/script/api/script_error.hpp

```diff
@@ -11,6 +11,7 @@
 #define SCRIPT_ERROR_HPP
 
 #include "script_object.hpp"
+#include "../../string_func.h"
```

Comment:
what's this doing here?

Reply:
Well, it kinda uses StrEmpty from string_func.h without including it directly.
It's in #define macro though so not sure if should be included here at all.
Also I ended up including string_func.h in script_industry.cpp anyway so it's not even needed for compilation.

Reply:
Yeah, not really. Defines are, by definition, not actually using anything from other files